client/Pages/SubscriptionCreate/Update.elm

```diff
@@ -359,25 +417,43 @@ searchEvenType eventTypeStore filter =
             results
 
 
+updateSubscription : Stores.Subscription.Model -> String -> Model -> Model
+updateSubscription subscriptionStore id model =
+    let
+        maybeSubscription =
+            Store.get id subscriptionStore
+
+        values =
+            case maybeSubscription of
+                Just subscription ->
+                    copyValues subscription
+
+                Nothing ->
+                    initialModel.values
+    in
+        { initialModel | values = values, cursorsStore = model.cursorsStore, operation = model.operation }
+
+
 cloneSubscription : Stores.Subscription.Model -> String -> Model -> Model
 cloneSubscription subscriptionStore id model =
     let
         maybeSubscription =
             Store.get id subscriptionStore
 
-        cursors =
-            model.cursorsStore
-                |> Store.items
-                |> List.map Stores.Cursor.subscriptionCursorEncoder
-                |> Json.list
-                |> Json.encode 1
-
         values =
             case maybeSubscription of
                 Just subscription ->
-                    copyValues cursors subscription
+                    cloneValues subscription
 
                 Nothing ->
                     initialModel.values
     in
-        { initialModel | values = values, cursorsStore = model.cursorsStore, cloneId = model.cloneId }
+        { initialModel | values = values, cursorsStore = model.cursorsStore, operation = model.operation }
+
+
+authorizationFromSubscription : Maybe String -> Stores.Subscription.Model -> Authorization
+authorizationFromSubscription maybeId subscriptionsStore =
+    maybeId
+        |> Maybe.andThen (\id -> Store.get id subscriptionsStore)
+        |> Maybe.andThen .authorization
+        |> Maybe.withDefault emptyEventTypeAuthorization
```

Comment:
It seems a bit strange to have EventTypeAuthorization in Subscription

Reply:
the same.
It is now just general "Authorization". Probably it is better to rename it.